index.js

```diff
@@ -0,0 +1,18 @@
+console.log("hello my friend");
```

Comment:
- [ ] Remove Console Logs, Console logs aren't good to have on production files, and should only be used for debugging, since you're not expected to use JavaScript on this section, the rules aren't listed, but you'll learn in the future why we shouldn't be using them in the projects. 

Awesome work though on the JQuery, (though I advice you to use DOM instead, [here's why](https://stackoverflow.com/questions/10711823/jquery-vs-dom-performance/10712102))